scripts/deploy.sh

```diff
@@ -14,7 +19,7 @@ cargo publish
 echo "Authenticating to npmjs.org registry..."
 echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > ~/.npmrc
 
-ci/build-wasm.sh
+$HERE/build-wasm.sh
```

Comment:
Will fail if `$HERE` will contain space for example. Use `"${HERE}/build-wasm.sh"`.